include/oneapi/mkl/exceptions.hpp

```diff
@@ -115,6 +115,14 @@ class batch_error : public oneapi::mkl::exception {
                                      "batch error" + ((info.length() != 0) ? (": " + info) : "")) {}
 };
 
+class library_not_found : public oneapi::mkl::exception {
+public:
+    library_not_found(const std::string &domain, const std::string &function,
+                const std::string &info = "")
```

Comment:
(minor): this indentation looks wrong, did you clang-format this file?

Reply:
Good catch, thank you!